MI-223 Implement : Save account data on login

NassimElSayed opened the pull request:
Only the MainActivity was modified. When a user opens the application for the first time after installing it, it will ask to login with facebook with the credentials. After this, it will automatically log the user with that account. The only issue right now is that there is no way to change the account (log out), so I will create an issue to add this functionality. 

I added a method handleRequest(Account account), which was the code inside the method OnAuthenticationCompleted. It allows to make a request from onCreate() and OnAuthenticationCompleted() without duplicating all the lines.

NassimElSayed committed:
MI-223 Implemented save account data on login to allow users to automatically login the next time

Ideawin commented on mRides-app/MainActivity.cs:
```diff
@@ -142,13 +148,24 @@ protected override void OnResume()
             base.OnCreate(bundle);
-            SetContentView(Resource.Layout.Main);
+            var account = AccountStore.Create(this).FindAccountsForService("Facebook").FirstOrDefault();
+            //IEnumerable<Account> accounts = AccountStore.Create(this).FindAccountsForService("Facebook");
```
Is this commented line of code necessary since we have the one above? If not, it is better to remove it in case you will forget.

NassimElSayed replied:
You are right bad habit i'll remove it

NassimElSayed committed:
MI-223 Added splash screen when the user uses the saved login information.

I added splash.xml  and styles.xml that contains the UI part of the splash screen. SplashActivity is now the MainLauncher and I moved the method handleRequest from MainActivity to a new class called LoginRequest. SplashActivity and MainActivity uses the method handleRequest

NassimElSayed committed:
Merged dev into Maps

AlineTheAlien commented:
Looks good, merging.
